infra/build/functions/update_build_status.py

```diff
@@ -184,14 +204,32 @@ def update_build_badges(project, last_build_successful,
         project_name=project,
         extension=extension)
 
-    status_bucket = builds_status.get_storage_client().get_bucket(
-        builds_status.STATUS_BUCKET)
+    status_bucket = get_storage_client().get_bucket(STATUS_BUCKET)
     badge_blob = status_bucket.blob(blob_name)
     status_bucket.copy_blob(badge_blob,
                             status_bucket,
                             new_name=destination_blob_name)
 
 
+def upload_log(build_id):
+  """Uploads log file oss-fuzz-build-logs."""
```

Comment:
super nit: "upload **to** oss-fuzz-build-logs" or maybe just "to GCS"

Reply:
Done!